docs/actions-scripts.md

```diff
@@ -31,10 +31,35 @@ These Docker images have yet to be optimised; if you have skills in creating Doc
 
 We currently package version 16.1, with `datacheck`, `safetab`, and `safecount` libraries installed; when installed, new libraries will appear [in the stata-docker GitHub repository](https://github.com/opensafely-core/stata-docker/tree/master/libraries).
 
-As Stata is a commercial product, a license key is needed to use it. If you are
-a member of the opensafely GitHub organisation, then the tooling will
-automatically use the OpenSAFELY stata license. If not, get in touch if you
-need to apply your own license and we can help.
+As Stata is a commercial product, a license key is needed to use it. 
+
+#### If you are a member of the opensafely GitHub organisation
+* If you are using Windows, then the `opensafely` command line software will
+automatically use the OpenSAFELY stata license. 
+* If you are using macOS:
+   * Download and install the [github cli tool (`gh`)](https://cli.github.com/)
+   * Run `gh auth login --web`. Select the "HTTPS" option, and follow the instructions
+  * The `opensafely` command line software will now automatically use the OpenSAFELY stata license
+
+#### All other external users
+
+If you are not a member of the opensafely GitHub organisation, you will need to provide your own Stata-MP license. Unfortunately other flavours of Stata are not yet supported; let us know if this is a problem.
+
+First, locate your Stata license string as follows:
+
+Locate a text file, called STATA.LIC (on Windows) or stata.lic (macOS and Linux) which is usually at the top level of the folder of your Stata installation:
+
+* On Windows machine it's usually somewhere like `C:\Program Files\Stata17`
+* On Linux, somewhere like `/usr/local/stata17/`
+* On macOS it's usually in `/Applications/Stata/`
+
+Within that file, you'll find a license string of the format `SerialNumber!Code!Authorization!User!Organisation!VersionCode`. 
+
+Now set it as an environment variable, for example:
+
+    export STATA_LICENSE='your license string'
+
+The `opensafely` command line software will now automatically use this OpenSAFELY stata license.
```

Comment:
nit:
```suggestion
The `opensafely` command line software will now automatically use this Stata license.
```

Reply:
Also maybe add a line saying:

You will need to repeat this step each time you open a new terminal.